examples/ae/invoice-ae-ae-stnr.yaml

```diff
@@ -0,0 +1,48 @@
+$schema: "https://gobl.org/draft-0/bill/invoice"
```

Comment:
I can't see the output from these files, did you run `go generate ./...`?

Reply:
I have implemented the JSON output for the examples. However, I had to remove the Arabic version of the notes in scenario.go because it wasn't generating the Arabic characters correctly.

Below is how the notes appeared originally:
![gobl-ae-scenarios go](https://github.com/user-attachments/assets/b01312fe-ab8c-44b1-8271-5d14a32a4b94)


And here is how the incorrect characters were generated:
![gobl-ae-example-invoice go](https://github.com/user-attachments/assets/ea2733d5-1082-4123-a676-8e6df7b3c2e1)